### PR TITLE
chore: use set instead of export in windows batch

### PR DESCRIPTION
**Pull request by jflatt-gia**
Fixes: Build complains about export command

### What was the problem/requirement? (What/Why)
Test phase under win-64 platform uses the batch file and complains about export command

### What was the solution? (How)
Use set instead of export in windows batch

### What is the impact of this change?
Environment variables are set

### How was this change tested?
1. Build package under win-64, observe test phase complains about export, build succeeds anyways
2. Change export to set
3. Build package again, output is cleaner

- If this is a sample, then please describe the steps that you took to test it.
- Include output from your testing to demonstrate it working as expected if possible.

### Was this change documented?
Not applicable

- For instance, if applicable, has the sample's description been updated?

---

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

**Comment by crowecawcaw** [comment]
We have a couple different patterns for conda build scripts in our samples and packages. In some cases for Windows, we've made `.bat` files that are meant to be run as Windows batch files. In some cases like this one, we've made a `.sh` file for Windows that's intended to be run with Git Bash for Windows. Using bash for both Linux and Windows scripts makes it a bit easier to support platforms at the same time.

So TLDR: I think these commands are correct and should be run with Git Bash.

**Review by mwiebe — APPROVED** [review]

**Comment by mwiebe** [comment]
Reviewing this older PR, it looks like it is correct and we should apply it. The change is modifying the .bat files produced. Apologies for the confusion and delays in getting this in!

**Review by eherozhao — APPROVED** [review]
